lib/node_modules/@stdlib/array/fixed-endian-factory/README.md

```diff
@@ -493,6 +493,22 @@ var arr = new Float64ArrayFE( 'little-endian', [ 1.0, 2.0, 3.0 ] );
 var str = arr.toString();
 // returns '1,2,3'
 ```
+#### TypedArrayFE.prototype.slice()
```

Comment:
See https://github.com/stdlib-js/stdlib/tree/4b1d77d2bd001d5970ce93230765a579fb41349d/lib/node_modules/%40stdlib/array/bool#booleanarrayprototypeslice-start-end- for what is excepted in terms of documentation. Additionally, this documentation should be inserted such that methods are listed in alphabetical order.